pkg/api/service/finding.go

```diff
@@ -87,11 +88,10 @@ func isValidFindingTransition(status, statusPrevious string) bool {
 		return true
 	}
 
-	validTransitions := map[string]any{
-		"OPEN->FALSE_POSITIVE":  nil,
-		"FALSE_POSITIVE->OPEN":  nil,
-		"FIXED->FALSE_POSITIVE": nil,
+	validTransitions := map[string][]string{
+		"OPEN":           []string{"FALSE_POSITIVE"},
+		"FALSE_POSITIVE": []string{"OPEN"},
+		"FIXED":          []string{"FALSE_POSITIVE"},
```

Comment:
You can do what you did in https://github.com/adevinta/vulnerability-db-api/pull/89:

```suggestion
		"OPEN":           {"FALSE_POSITIVE"},
		"FALSE_POSITIVE": {"OPEN"},
		"FIXED":          {"FALSE_POSITIVE"},
```

It looks cleaner.

Reply:
Have you missed this one or do you prefer to discard the suggestion?

Reply:
Sorry I missed, many RPs at the same time :(